### ISSUE TITLE
how!

### ISSUE THREAD

**Issue by MahmoudKhaiti** [scheng123321/1v1-lol#3]
How can I insert it?

**Comment by CuriousCoder1232** [comment]
u need a webstite

**Comment by MahmoudKhaiti** [comment]
wdym?

**Comment by 3kh0** [comment]
@MahmoudKhaiti You need google firebase, it is very complex, just use the official one on the google site

**Comment by MahmoudKhaiti** [comment]
bruh can u pls record ur screen so i can see how exactly

**Comment by 3kh0** [comment]
@MahmoudKhaiti It is not something I can just show you. Plus, you have to pay for the google firebase service. You will need to know how to use a FTP and know how to operate servers through the UI and console

**Comment by MahmoudKhaiti** [comment]
ya know what, i think im ok not cheating in it, lemme just play it normaly

**Comment by NotSorena** [comment]
use github pages, its really easy to use for beginners

**Comment by lmaopolar** [comment]
> [@MahmoudKhaiti](https://github.com/MahmoudKhaiti) It is not something I can just show you. Plus, you have to pay for the google firebase service. You will need to know how to use a FTP and know how to operate servers through the UI and console

That's false. You do not need to pay for the google firebase service, anymore. It depends on your usage and maximum rates. Unfortunately, you also won't be getting 2k+ players like you'd see on a skidded game site, they use a different firebase that's publicly shared, that they've most likely gotten granted access to by someone on the inside who controls it. Firestore is locked one on one to a specific site, using x-origin headerlock, stopping it from being embedded, or used on the site it shouldn't be on. People are most likely pardoned however, because they're able to implement that same 1v1.lol into their site.
